externs/olx.js

```diff
@@ -3104,6 +3104,15 @@ olx.interaction.PinchZoomOptions;
  */
 olx.interaction.PinchZoomOptions.prototype.duration;
 
+/**
+ * `true` to zoom to next whole-number zoom after the pinch completes;
+ * `false` to keep the zoom at an fractional number.
+ * Default is `false`.
```

Comment:
When documenting booleans, just describe the behavior when true (and then identify the default).  For example, this could be:

> Zoom to the closest integer zoom level after the pinch gesture ends.  Default is `false`.